hydroshare/urls.py

```diff
@@ -30,6 +31,7 @@
     # admin interface, which would be marginally more secure.
     url("^admin/", include(admin.site.urls)),
     url(r'^o/', include('oauth2_provider.urls', namespace='oauth2_provider')),
+    url("^o/groupauthorize/(?P<group_id>[0-9]+)/$", GroupAuthorizationView.as_view(), name="group-authorize"),
```

Comment:
If passing the group id in the url is a security flaw, we should be able to extend Application and register a group id there.